config/environment.js

```diff
@@ -128,7 +128,7 @@ module.exports = function(environment) {
 
     SAMPLE_SELECTION,
 
-    SupportServiceHost: 'https://factfinder-api-staging.planninglabs.nyc',
+    SupportServiceHost: 'https://factfinder-api-staging.herokuapp.com',
```

Comment:
Is this just to make it more clear the exact host?

Reply:
the planninglabs.nyc url doesn't work anymore so it's a change to fix the testing